ci/test_cpp.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# TODO: Move this
+. /opt/conda/etc/profile.d/conda.sh
+conda activate base
```

Comment:
same question.

do we still need to move this? if not, can we remove this comment?

looks like there's a few other instances of this in the PR as well.

Reply:
should we move this to a `rapids-activate-base-env` script in `gha-tools`?